server/ServerApp.cpp

```diff
@@ -318,7 +318,6 @@ int ServerApp::GetNewDesignID()
 
 void ServerApp::Run() {
     DebugLogger() << "FreeOrion server waiting for network events";
-    std::cout << "FreeOrion server waiting for network events" << std::endl;
```

Comment:
It's useful for attaching the debugger to catch something before whatever state the server is in on the galaxy setup screen.